File: rxjava/src/main/java/com/asemlab/samples/rxjava/models/CountryResponseItem.java
package com.asemlab.samples.rxjava.models;

import androidx.annotation.Keep;
import com.google.gson.annotations.SerializedName;

import java.io.Serializable;


@Keep
// TODO Make the response classes Serializable if don't use Gson
//@Serializable
public class CountryResponseItem implements Serializable {

    @SerializedName("flags")
    public Flags flags;

    @SerializedName("name")
    public Name name;
    @SerializedName("region")
    public String region;

}

File: ktor/src/main/java/com/asemlab/samples/ktor/models/CountryResponseItem.kt
package com.asemlab.samples.ktor.models


import androidx.annotation.Keep
import com.google.gson.annotations.SerializedName

@Keep
// TODO Make the response classes Serializable if don't use Gson
//@Serializable
data class CountryResponseItem(
    @SerializedName("flags")
    val flags: Flags? = null,
    @SerializedName("name")
    val name: Name? = null,
    @SerializedName("region")
    val region: String? = null
)